libs/net/entities/Source.cs

```diff
@@ -45,6 +45,17 @@ public class Source : BaseType<int>
     /// </summary>
     public virtual User? Owner { get; set; }
 
+    /// <summary>
+    /// get/set - Foreign key to product that content will be defaulted to.
+    /// </summary>
+    [Column("product_id")]
+    public int? ProductId { get; set; }
```

Comment:
@sturufous I've added the property to the table, so you shouldn't need to worry about updating the Initial migration.  You will still need to update all the relevant models.